app/models/concerns/foreman_openscap/hostgroup_extensions.rb

```diff
@@ -16,6 +16,15 @@ def inherited_policies
       end.uniq
     end
 
+    def openscap_proxy
+      return super if ancestry.nil? || self.openscap_proxy_id.present?
+      ::SmartProxy.find_by(:id => inherited_openscap_proxy_id)
+    end
+
+    def inherited_openscap_proxy_id
+      inherited_ancestry_attribute(:openscap_proxy_id)
```

Comment:
This method is for some reason only defined `unless defined?(Katello::System)`, if we want to use it here, we should make it always available. It seems that you added it some time ago, do you remember why the condition was required?